src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- Hey, please, let's try to write codes that conform to the linter as this would help improve our development skill. please, let's remove this eslint disable and fix the linter.

Reply:
Hi, @zieeco, I have attached a link to an issue where I explain my [reasoning](https://github.com/Timbar09/React-Bookstore/issues/2) for disabling this linter rule. I'm not sure if you have seen it.

Can you confirm?